aws/resource_aws_instance_test.go

```diff
@@ -530,6 +548,12 @@ func TestAccAWSInstance_blockDevices(t *testing.T) {
 						"volume_type": "io1",
 						"iops":        "100",
 					}),
+					resource.TestCheckTypeSetElemNestedAttrs(resourceName, "ebs_block_device.*", map[string]string{
+						"device_name": "/dev/sde",
+						"volume_size": "10",
+						"volume_type": "gp3",
+						"throughput":  "1000",
+					}),
```

Comment:
Acceptance test failure:

```
=== CONT  TestAccAWSInstance_blockDevices
    resource_aws_instance_test.go:521: Step 1/2 error: Check failed: Check 10/16 error: "aws_instance.test" no TypeSet element "ebs_block_device.*", with nested attrs map[string]string{"device_name":"/dev/sde", "throughput":"1000", "volume_size":"10", "volume_type":"gp3"} in state: map[string]string{"%":"45", "ami":"ami-094f9475eb34352bd", "arn":"arn:aws:ec2:us-west-2:123456789012:instance/i-03a98f64ee5b924c4", "associate_public_ip_address":"true", "availability_zone":"us-west-2a", "cpu_core_count":"2", "cpu_threads_per_core":"1", "credit_specification.#":"1", "credit_specification.0.%":"1", "credit_specification.0.cpu_credits":"standard", "disable_api_termination":"false", "ebs_block_device.#":"3", "ebs_block_device.0.%":"10", "ebs_block_device.0.delete_on_termination":"true", "ebs_block_device.0.device_name":"/dev/sdb", "ebs_block_device.0.encrypted":"false", "ebs_block_device.0.iops":"100", "ebs_block_device.0.kms_key_id":"", "ebs_block_device.0.snapshot_id":"", "ebs_block_device.0.throughput":"0", "ebs_block_device.0.volume_id":"vol-033d3dc5de12a8b8f", "ebs_block_device.0.volume_size":"9", "ebs_block_device.0.volume_type":"gp2", "ebs_block_device.1.%":"10", "ebs_block_device.1.delete_on_termination":"true", "ebs_block_device.1.device_name":"/dev/sdc", "ebs_block_device.1.encrypted":"false", "ebs_block_device.1.iops":"100", "ebs_block_device.1.kms_key_id":"", "ebs_block_device.1.snapshot_id":"", "ebs_block_device.1.throughput":"0", "ebs_block_device.1.volume_id":"vol-0c46a050c0e65b8a7", "ebs_block_device.1.volume_size":"10", "ebs_block_device.1.volume_type":"io1", "ebs_block_device.2.%":"10", "ebs_block_device.2.delete_on_termination":"true", "ebs_block_device.2.device_name":"/dev/sdd", "ebs_block_device.2.encrypted":"true", "ebs_block_device.2.iops":"100", "ebs_block_device.2.kms_key_id":"arn:aws:kms:us-west-2:123456789012:key/de1b833b-bc4c-4ce6-836a-67ce09375aca", "ebs_block_device.2.snapshot_id":"", "ebs_block_device.2.throughput":"0", "ebs_block_device.2.volume_id":"vol-039d17b0c624d94f5", "ebs_block_device.2.volume_size":"12", "ebs_block_device.2.volume_type":"gp2", "ebs_optimized":"false", "ephemeral_block_device.#":"1", "ephemeral_block_device.0.%":"3", "ephemeral_block_device.0.device_name":"/dev/sde", "ephemeral_block_device.0.virtual_name":"ephemeral0", "get_password_data":"false", "hibernation":"false", "iam_instance_profile":"", "id":"i-03a98f64ee5b924c4", "instance_state":"running", "instance_type":"t2.medium", "ipv6_address_count":"0", "ipv6_addresses.#":"0", "key_name":"", "metadata_options.#":"1", "metadata_options.0.%":"3", "metadata_options.0.http_endpoint":"enabled", "metadata_options.0.http_put_response_hop_limit":"1", "metadata_options.0.http_tokens":"optional", "monitoring":"false", "network_interface.#":"0", "outpost_arn":"", "password_data":"", "placement_group":"", "primary_network_interface_id":"eni-09552b92cc8fffd8b", "private_dns":"ip-172-31-18-32.us-west-2.compute.internal", "private_ip":"172.31.18.32", "public_dns":"ec2-34-220-195-26.us-west-2.compute.amazonaws.com", "public_ip":"34.220.195.26", "root_block_device.#":"1", "root_block_device.0.%":"9", "root_block_device.0.delete_on_termination":"true", "root_block_device.0.device_name":"/dev/xvda", "root_block_device.0.encrypted":"false", "root_block_device.0.iops":"100", "root_block_device.0.kms_key_id":"", "root_block_device.0.throughput":"0", "root_block_device.0.volume_id":"vol-010803ff5509b9781", "root_block_device.0.volume_size":"11", "root_block_device.0.volume_type":"gp2", "secondary_private_ips.#":"0", "security_groups.#":"1", "security_groups.0":"default", "source_dest_check":"true", "subnet_id":"subnet-e26cd99b", "tenancy":"default", "volume_tags.%":"0", "vpc_security_group_ids.#":"1", "vpc_security_group_ids.0":"sg-e7bc7699"}
--- FAIL: TestAccAWSInstance_blockDevices (66.09s)
```

Requires modifying `testAccAwsEc2InstanceConfigBlockDevices()/testAccAwsEc2InstanceConfigBlockDevicesWithDeleteOnTerminate()`.

Reply:
Ok, I think I fixed it in ad5f540